activestash.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.metadata["source_code_uri"] = "https://github.com/cipherstash/activestash"
   spec.metadata["changelog_uri"] = "https://github.com/cipherstash/activestash/CHANGELOG.md"
 
-  spec.add_runtime_dependency "cipherstash-client", ">= 0.10.2"
+  spec.add_runtime_dependency "cipherstash-client", ">= 0.10.4"
```

Comment:
This is unnecessary -- bundler will automatically install the latest available version of a gem that otherwise meets the version constraints on the dependency.

Reply:
@mpalmer I tried a `bundle install` locally and it ... didn't.

Reply:
But the build is _broken_ for Rubies < 0.10.4 so we should specify the minimum working version, right?